encoders/image/TirgImageEncoder/README.md

```diff
@@ -1,3 +1,64 @@
 # TirgImageEncoder
 
-`TirgImageEncoder` encodes data from a ndarray, potentially B x (Channel x Height x Width) into a B x d ndarray
\ No newline at end of file
+**TIRG** model was original proposed in [Composing Text and Image for Image Retrieval - An Empirical Odyssey](https://arxiv.org/abs/1812.07119).
+It has been used in Jina multimodal retrieval example.
+
+During the indexing time, the **TIRG** model takes `TirgImageEncoder` and encode images into embeddings.
+At query time, the **TIRG** model takes `TirgMultimodalEncoder` and encode images together with it's associated texts into embeddings.
+
+As a part of **TIRG** model,
+`TirgImageEncoder` encodes data from a ndarray, potentially B x (Channel x Height x Width) into a B x d ndarray.
```

Comment:
what is B ? (batch size?) if so it should be defined

Reply:
got it, thanks!